xml/System/Random.xml

```diff
@@ -827,7 +829,7 @@ The following example uses the parameterless constructor to instantiate three <x
   
```

Comment:
This page is also missing the note. But in this case you can just add under the Examples header since there are no other overloads.

Reply:
Done.